src/mumble/ServerHandler.cpp

```diff
@@ -59,7 +59,7 @@
 
 // Init ServerHandler::nextConnectionID
 int ServerHandler::nextConnectionID = -1;
-QMutex ServerHandler::nextConnectionIDMutex(QMutex::Recursive);
+QMutex ServerHandler::nextConnectionIDMutex;
```

Comment:
This seems wrong, it was using Recursive mode unlike above. Are you sure this mutex didn't need recursive mode? @Krzmbrzl

Reply:
Oh lol. You are right :eyes:

Reply:
In this particular case however it doesn't matter since the mutex should have been non-recursive anyways. It is only ever used in a single place (constructor of `ServerHandler`) and therefore it does not have to be recursive.
That was probably an oversight from the initial implementation that was now accidentally fixed xD

Thanks for pointing this out though! :+1:

Reply:
Damn, I saw that and got confused between diffs and files and local, and somehow lost that train of thought/verifying. lol